planning/behavior_path_planner/include/behavior_path_planner/scene_module/pull_out/pull_out_parameters.hpp

```diff
@@ -0,0 +1,53 @@
+
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BEHAVIOR_PATH_PLANNER__SCENE_MODULE__PULL_OUT__PULL_OUT_PARAMETERS_HPP_
+#define BEHAVIOR_PATH_PLANNER__SCENE_MODULE__PULL_OUT__PULL_OUT_PARAMETERS_HPP_
+
+namespace behavior_path_planner
+{
+struct PullOutParameters
+{
+  double th_arrived_distance_m;
+  double th_stopped_velocity_mps;
+  double th_stopped_time_sec;
+  double collision_check_margin;
+  double pull_out_finish_judge_buffer;
+  // shift pull out
+  bool enable_shift_pull_out;
+  double shift_pull_out_velocity;
+  int pull_out_sampling_num;
+  double before_pull_out_straight_distance;
+  double minimum_shift_pull_out_distance;
+  double maximum_lateral_jerk;
+  double minimum_lateral_jerk;
+  double deceleration_interval;
+  // geometric pull out
+  bool enable_geometric_pull_out;
+  double geometric_pull_out_velocity;
+  double arc_path_interval;
+  double lane_departure_margin;
+  double backward_velocity;
+  double max_steer_rad;
```

Comment:
write the reason not using vehicle_info's in README

Reply:
pull_out_max_steer_angle

Reply:
renamed in https://github.com/autowarefoundation/autoware.universe/pull/1438/commits/47013746bacac7f1a66d9076c37ee483588bef69

Reply:
will update README in https://github.com/autowarefoundation/autoware.universe/pull/1587